src/libraries/System.Private.Xml/src/System/Xml/Schema/XsdDateTime.cs

```diff
@@ -396,6 +396,7 @@ InternalKind switch
             {
                 case DateTimeTypeCode.GMonth:
                 case DateTimeTypeCode.GDay:
+                    // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime not constructed from multiple elements - it's converted from an XsdDateTime.
```

Comment:
Maybe say it is intentional to throw on parsed date with 2/29 on non-leap years.

Reply:
```suggestion
                    // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime not constructed from multiple elements - it's converted from an XsdDateTime.  It would be intentional to throw if an invalid combination occurred.
```